assets/js/util/test/numberFormat.js

```diff
@@ -87,6 +87,7 @@ describe( 'numberFormat', () => {
 		expect(
 			numberFormat( 123456789.87, { locale: 'de-DE' } )
 		).toStrictEqual( '123.456.789,87' );
+		expect( console ).not.toHaveWarned();
```

Comment:
Nice touch 👍  is it possible to add a test or two that will trigger the fallback as well?

Reply:
Yep, addressed in https://github.com/google/site-kit-wp/pull/3620/commits/e07d41e9a513daa7c1751bdba7105e69dfc707a9